DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -149,7 +149,8 @@
    <version ClassVersion="10" checksum="2692173055"/>
   </class>
   <class name="std::vector<pat::tau::TauCaloSpecific>" />
-  <class name="pat::tau::TauPFEssential" ClassVersion="11">
+  <class name="pat::tau::TauPFEssential" ClassVersion="12">
+    <version ClassVersion="12" checksum="1052833547" />
```

Comment:
Ok I've added now the intermediate classVersion (as classVersion 12) as it is 81X